.github/workflows/ci.yml

```diff
@@ -22,5 +22,15 @@ jobs:
     - name: Build with Gradle
       run: cd extractor &&  ./gradlew clean build
 
+    - name: Remove existing config folders
+      run: |
+        cd extractor
+        find . -maxdepth 1 -type d -name '[A-Z]*' -exec rm -rf {} +
```

Comment:
No .
Such folder will have manually created files, you can remove only files extractor is generating.
You can run this command your self few times for now. But not good for CI